src/components/map/MapService.js

```diff
@@ -874,12 +874,12 @@
           }
 
           if (!angular.isDefined(yearStr)) {
-            var timeBehaviour = this.getLayerProperty(bodId,
-                'timeBehaviour');
-            yearStr = (timeBehaviour === 'all' || timestamps.length == 0) ?
-                undefined : timestamps[0];
-            if (bodId == 'ch.swisstopo.zeitreihen') {
-              yearStr = '18641231';
+            var timeBehaviour = this.getLayerProperty(bodId, 'timeBehaviour');
+            //check if specific 8 digit timestamp is specified
+            if (/^\d{8}$/.test(timeBehaviour)) {
```

Comment:
In case of a WMS layer like `ch.swisstopo.lubis-bildstreifen`  tiemstamps are 4 digits